app/src/main/java/com/example/trempel/ui/products_in_category/CategoryProductsRecyclerAdapter.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.trempel.ui.products_in_category
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.trempel.databinding.CategoryProductsItemBinding
+import com.example.trempel.network.model.CategoryModelItem
+
+class CategoryProductsRecyclerAdapter :
```

Comment:
I wonder if it can be marked as `internal`

Reply:
done